internal/command/versions.go

```diff
@@ -0,0 +1,48 @@
+package command
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+
+	"github.com/cyberark/summon/pkg/summon"
+	"github.com/cyberark/summon/provider"
+)
+//Create the AllVersions string to include versions of all the providers
+const SUMMON_VERSION = summon.VERSION
+var DefaultPath = provider.DefaultPath
+var AllVersions = GetVersions(GetProviderFileNames())
```

Comment:
Normally we avoid doing this kind of thing as there's usually no need to preserve a variable for a one-shot evaluation (and it often leads to less readable code).